backend/packages/Upgrade/src/api/services/StratificationService.ts

```diff
@@ -220,4 +224,18 @@ export class StratificationService {
 
     return createdStratificationData;
   }
+
+  public async insertStratificationFiles(
+    files: UploadedFilesValidator[],
+    logger: UpgradeLogger
+  ): Promise<UserStratificationFactor[]> {
+    const stratificationResults: UserStratificationFactor[] = [];
```

Comment:
This could be refactored with Array.map
return files.map((file) => this.insertStratification(fileObj.file, logger))

Reply:
I can handle this ticket now.